pkg/cedar/core/system_summary.go

```diff
@@ -69,14 +38,17 @@ func (c *Client) GetSystemSummary(ctx context.Context, tryCache bool, opts ...sy
 
 	// set additinoal param filters
 	for _, opt := range opts {
-		opt(params)
+		if opt != nil {
```

Comment:
did you run into a `nil` opt?

Reply:
Ah, yes. Justin and I ran into this when developing where the ATO page wouldn't load for a `nil` dereference here. There was a slice of `nil` being passed in, so I added an if.

Reply:
weird! i wonder how that happened - thanks for letting me know